src/coreclr/System.Private.CoreLib/src/System/Reflection/Metadata/AssemblyExtensions.cs

```diff
@@ -40,5 +40,54 @@ public static unsafe bool TryGetRawMetadata(this Assembly assembly, out byte* bl
 
             return InternalTryGetRawMetadata(new QCallAssembly(ref rtAsm), ref blob, ref length);
         }
+
+        [DllImport(RuntimeHelpers.QCall)]
+        private static extern unsafe int ApplyHotReloadUpdate(QCallAssembly assembly, byte* metadataDelta, int metadataDeltaLength, byte* ilDelta, int ilDeltaLength, byte* pdbDelta, int pdbDeltaLength);
```

Comment:
Can we remove "HotReload" from the naming?

Reply:
Done.